poetry.lock

```diff
@@ -1145,288 +1145,290 @@ s3transfer = ">=0.3.0,<0.4.0"
 
 [[package]]
 name = "boto3-stubs"
-version = "1.17.61.0"
-description = "Type annotations for boto3 1.17.61, generated by mypy-boto3-buider 4.7.0"
+version = "1.17.71"
+description = "Type annotations for boto3 1.17.71, generated by mypy-boto3-buider 4.12.0"
 category = "dev"
 optional = false
 python-versions = ">=3.6"
 
 [package.dependencies]
-mypy-boto3-batch = {version = "1.17.61.0", optional = true, markers = "extra == \"batch\""}
```

Comment:
I don't think we want to upgrade these packages as part of this change. One "simple" way to work around this would be to

0. Revert back to before this change
1. `poetry lock --no-update`
2. `git commit -m 'chore: Throw-away commit'`
3. `poetry add whatever`
4. `git commit`
5. `git rebase` and remove the throw-away commit